packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container.jsx

```diff
@@ -97,7 +97,17 @@ const ProofOfIdentityContainer = ({
     }
 
     const redirect_button = should_show_redirect_btn && (
-        <Button primary className='proof-of-identity__redirect' onClick={() => routeBackTo(from_platform.route)}>
+        <Button
+            primary
+            className='proof-of-identity__redirect'
+            onClick={() => {
+                if (platforms[from_platform.name?.toLowerCase()].is_hard_redirect) {
+                    window.location.href = platforms[from_platform.name?.toLowerCase()].url;
```

Comment:
This pattern may cause security issues, please change it like this :pray: 
```suggestion
const url = platforms[from_platform.name?.toLowerCase()].url;
window.location.href = url;
```

Reply:
More details: https://codeql.github.com/codeql-query-help/javascript/js-xss-through-dom/

Reply:
updated